### PR TITLE
Frivolities

### PR DESCRIPTION
**Pull request by Ambrevar**
@BlueFlo0d Excellent work, this is really impressive!
It will be a great improvement over the existing hook system in Nyxt!

I've addressed a bunch of frivolities in this patch.  Can you have a look and let me know what you think?

### CODE REVIEW

**Comment by BlueFlo0d** [comment]
Thanks! I've added some frivolities on frivolities. After they're fixed it should be all good.

**Comment by Ambrevar** [comment]
Sorry, what have you added?

**Comment by BlueFlo0d** [comment]
> Sorry, what have you added?

Are the "reviews" visible to you? Just 2 minor fixes.

**Comment by Ambrevar** [comment]
They are not, you should press the "Submit review" button (or something like that).

File: nhooks.lisp [hunk 1]
```diff
@@ -43,7 +43,6 @@
    #:handler-type
    #:handlers-alist
    #:handlers
-   #:disabled-handlers
```
[the remark this answers is not in the document]
Comment (Ambrevar) [reply nhooks.lisp:46]: My bad, I missed it when I searched.

File: nhooks.lisp [hunk 2]
```diff
@@ -329,7 +329,8 @@ Without HANDLERS, enable all of them."
 
 (defun define-hook (hook-type name &key object handlers disabled-handlers combination)
   "Return a globally-accessible hook.
-The hook can be accessed with `find-hook' at (list NAME OBJECT)."
+The hook can be accessed with `find-hook' at (list NAME OBJECT).
+OBJECT is an arbitrary value the hook is associated too."
```
[the remark this answers is not in the document]
Comment (Ambrevar) [reply nhooks.lisp:332]: Good catch.

**Comment by Ambrevar** [comment]
Updated!

**Pull request merged**